less/home-page.less

```diff
@@ -1 +1,99 @@
-// Home Content
\ No newline at end of file
+// Home Content
+.home {
+   @media @phone{
+       width:100%;
+   }
+   
+    .middle-content{
+       margin-top:2.27%;
```

Comment:
Vertical margins don't necessarily _need_ to be percent based. Left and Right margins for sure, need to be.